.github/workflows/build_and_test.yml

```diff
@@ -885,7 +885,9 @@ jobs:
 
   deploy-new:
     runs-on: [ self-hosted, gen3, small ]
-    container: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/ansible:pinned
+    container:
+      image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/ansible:pinned
+      options: --user root --privileged
```

Comment:
What exactly requires `root`?

Reply:
Docker -> /etc/init.d/docker: 96: ulimit: error setting limit (Operation not permitted)

Reply:
I switched the user on that image to be "nonroot" else pipeline failures are painful to deal with.

Reply:
> Docker -> /etc/init.d/docker: 96: ulimit: error setting limit (Operation not permitted)

I'm a bit lost. Where do we run /etc/init.d/docker in this job?

Reply:
https://github.com/neondatabase/neon/actions/runs/4024470203/jobs/6917288451

in the https://github.com/neondatabase/neon/blob/main/.github/ansible/get_binaries.sh